packages/ssr/src/utils/chunker.ts

```diff
@@ -99,7 +99,6 @@ export async function deleteChunks(
 
 	if (value) {
 		await removeChunk(key);
-		return;
```

Comment:
removing the return here just makes this function delete all the chunks anyway - we should just remove this part and just keep the for loop below

Reply:
We need to delete the non-chunked and all potential chunks. Not just remove the non-chunked value.